sources/web/datalab/polymer/components/datalab-data/datalab-data.ts

```diff
@@ -31,6 +31,8 @@ class DataElement extends Polymer.Element {
 
   private _resultsList: Result[];
   private _searchValue: string;
+  private _searchInputDelayTimeout: number;
```

Comment:
This change to the way the search field behaves is completely orthogonal to the change for flattening the schema, and non-trivial. Please put in in a separate PR.